### ISSUE TITLE
winget install Microsoft.AzureCLI blocks install on hash verification failure

### ISSUE THREAD

**Issue by weikanglim** [Azure/azure-cli#13794]
**Describe the bug**
Installing Azure CLI through [winget](https://github.com/microsoft/winget-cli) currently shows warning due to hash verification failure on the downloaded installer.

**To Reproduce**
1. Install [winget](https://github.com/microsoft/winget-cli) 
2. Run `winget install -e Microsoft.AzureCLI`.
```
**PS C:\Users\USER> winget install -e Microsoft.AzureCLI
Found Azure CLI [Microsoft.AzureCLI]
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://aka.ms/installazurecliwindows
  ██████████████████████████████  53.6 MB / 53.6 MB
Installer hash verification failed. Continue? (Y|N)
Canceled; Installer hash mismatch**
```

A warning is displayed for the hash verification failing. If the user response with 'N', the install cancels as seen above.

winget manifest with SHA256 hash included:
```
PS C:\Users\USER> winget show -e Microsoft.AzureCLI
Found Azure CLI [Microsoft.AzureCLI]
Version: 2.6.0
Publisher: Microsoft Corporation
Description: The Azure command-line interface (Azure CLI) is a set of commands used to create and manage Azure resources. The Azure CLI is available across Azure services and is designed to get you working quickly with Azure, with an emphasis on automation.
Homepage: https://docs.microsoft.com/en-us/cli/azure/
License: Copyright (C) Microsoft Corporation
Installer:
  SHA256: b7fb439ab63dbea939d8f1d4ba2530dbb5939673e9450ce8ef5b55a71c22d68b
  Download Url: https://aka.ms/installazurecliwindows
  Type: Msi
```

**Expected behavior**
The `winget install` command should succeed without warnings.

**Environment summary**
n/a

**Additional context**
n/a

**Comment by weikanglim** [comment]
Judging by the downloaded file, the download URL is pointing towards version 2.7.0, but the manifest of the `winget` is outdated and still uses 2.6.0.

**Comment by yungezz** [comment]
hi @fengzhou-msft could you pls help to check if the url updated automatically?

**Comment by fengzhou-msft** [comment]
Submitted a [PR](https://github.com/microsoft/winget-pkgs/pull/1520) to fix 2.6.0 and a [PR](https://github.com/microsoft/winget-pkgs/pull/1516) to add 2.7.0.

**Comment by fengzhou-msft** [comment]
The latest version on winget is now 2.7.0.
